Dockerfile

```diff
@@ -1,12 +1,13 @@
 FROM docker:latest
+
 RUN apk add --no-cache git make bash python3 py3-pip && \
     pip3 install pyyaml
-WORKDIR "/app/"
+
+WORKDIR /app
+
 ENV XDG_CACHE_HOME=/app/cache
-ADD Docker/build/ Docker/build/
-ADD build/ build
+
+RUN mkdir /home/user && chmod 777 /home/user
```

Comment:
Would it make sense to give the user who executed the build process permissions to the artifacts after they're built (to avoid running as root in the container)?

Reply:
Thanks for catching the evil 777, applied the change we discussed in our call.